src/utp_stream.cpp

```diff
@@ -799,6 +802,23 @@ void utp_socket_drained(utp_socket_impl* s)
 	s->maybe_trigger_send_callback();
 }
 
+boost::int32_t adjust_receive_buf_size(boost::uint32_t const& rtt)
+{
+	// maximum value for RTT we would still scale for. We use it to clamp min_rtt 
+	// and to calculate the linear buffer increase coefficient
+	const boost::int32_t rtt_ceil = 800000;
```

Comment:
maybe the RTT type can be changed to use milliseconds here, to match the rest of the utp code (for RTT). In master I hope to make all these timestamps use chrono to fix type safety